docs/source/quickstart-docker.rst

```diff
@@ -1,11 +1,17 @@
 Install JupyterHub with Docker
 ==============================
 
+
+**important:**
+   We highly recommend following the `Zero to JupyterHub`_ tutorial for
+   installing JupyterHub.
+
 The JupyterHub `docker image <https://hub.docker.com/r/jupyterhub/jupyterhub/>`_ is the fastest way to set up Jupyterhub in your local development environment.
 
 The ``JupyterHub`` docker image runs the Hub service only. It does not provide other Jupyter components such as Notebook installation, which are needed by the single-user servers.
```

Comment:
sir @minrk do you think the notebook installation should be a link if there is a doc on it?

Reply:
Sure, you can link to https://jupyterlab.readthedocs.io